server/db/migrations/20200131013131-create-user.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Is there a reason the file name is prefixed with 20200131015659?

Reply:
It was autogenerated by sequelize-cli. I think it has meaning. These migration files have the sequence.

But I noticed this shouldn't be pushed to the repository. I will delete this.